python/jtmodules/src/jtmodules/expand_or_shrink.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Markus D. Herrmann, University of Zurich
+# Copyright 2016-2018 Markus D. Herrmann & Scott Berry, University of Zurich
```

Comment:
My understanding is that the *copyright* is held by the University (legal entity); you and Markus are the authors but do not hold the copy rights (since employed by the Uni, which is paying for this work).  But then, I'm no lawyer.

Reply:
I'm not sure, typically in academia the majority of IP (80-90%) is held by the "inventor(s)" rather than the institution. @hackermd, what is your understanding?

I don't mind where the authorship is mentioned as long at it is somewhere in the header.